db/migrate/20161123161514_create_samples_tables.rb

```diff
@@ -0,0 +1,33 @@
+class CreateSamplesTables < ActiveRecord::Migration
+  def change
+    create_table :samples_tables do |t|
+      t.jsonb :status, null: false,
+                       default: SampleDatatable::SAMPLES_TABLE_DEFAULT_STATE
+      # Foreign keys
+      t.references :user, null: false
+      t.references :organization, null: false
+
+      t.timestamps null: false
+    end
+    add_index :samples_tables, :user_id
+    add_index :samples_tables, :organization_id
+
+    User.find_each do |user|
+      next unless user.organizations
+      user.organizations.find_each do |org|
+        org_status = SampleDatatable::SAMPLES_TABLE_DEFAULT_STATE.deep_dup
+        next unless org.custom_fields
+        org.custom_fields.each_with_index do |_, index|
+          org_status['columns'] << { 'visible' => true,
+                                     'search' => { 'search' => '',
+                                                   'smart' => true,
+                                                   'regex' => false,
+                                                   'caseInsensitive' => true } }
```

Comment:
Same as before: can't you just use SampleDatatable::SAMPLES_TABLE_DEFAULT_STATE constant directly, without org_status?